openfl/utilities/utils.py

```diff
@@ -236,3 +236,27 @@ def merge_configs(
             settings.set(key, operation(value))
     settings.validators.validate()
     return settings
+
+
+def change_tags(tags, add_field=None, remove_field=None) -> Tuple[str]:
```

Comment:
We might need to add/remove multiple tags in a single operation. Does replacing string with lists make sense here?

Reply:
Yes but we did not find any good solution, maybe we could convert to class in future and reopen the issue.